scalableminds/webknossos
# Fixes taskType description formatting

### Mailable description of changes:
 - Fix the formatting of Task Type Descriptions

### Steps to test:
- Edit a task type to a longer Markdown-content with enumeration
```
1) Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. 
2) Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. 
          syn spine: Synapse onto a spine 
          syn shaft Synapse onto a shaft
          syn soma Synapse onto a soma
3) Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. 
4) Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. 
```
- Check TaskTypes and Dashboard-Task views
- Enumerations should be properly indented and column shouldn't be too wide


------
